pkg/sdk/testint/setup_integration_test.go

```diff
@@ -31,27 +32,59 @@ func setup() {
 	err := itc.initialize()
 	if err != nil {
 		log.Printf("Integration test context initialisation failed with %s\n", err)
+		cleanup()
 		os.Exit(1)
 	}
 }
 
 func cleanup() {
 	log.Println("Running integration tests cleanup")
+	if itc.databaseCleanup != nil {
+		defer itc.databaseCleanup()
+	}
 }
 
 type integrationTestContext struct {
 	client *sdk.Client
 	ctx    context.Context
+
+	database        *sdk.Database
+	databaseCleanup func()
 }
 
 func (itc *integrationTestContext) initialize() error {
 	log.Println("Initializing integration test context")
 	var err error
-	itc.client, err = sdk.NewDefaultClient()
+	c, err := sdk.NewDefaultClient()
+	if err != nil {
+		return err
+	}
+	itc.client = c
 	itc.ctx = context.Background()
+
+	db, dbCleanup, err := createDb(itc.client, itc.ctx)
+	itc.database = db
+	itc.databaseCleanup = dbCleanup
+
 	return err
 }
 
+func createDb(client *sdk.Client, ctx context.Context) (*sdk.Database, func(), error) {
+	u, err := uuid.GenerateUUID()
+	if err != nil {
+		return nil, nil, err
+	}
+	id := sdk.NewAccountObjectIdentifier("int_test_db_" + u)
+	err = client.Databases.Create(ctx, id, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	database, err := client.Databases.ShowByID(ctx, id)
```

Comment:
check the err or ignore with _ if you're not using it

Reply:
what do you mean? It is returned from the method.

Reply:
ah, you're right, didn't see it